src/improvements/template/TransferOwnerTemplate.sol

```diff
@@ -13,9 +13,9 @@ contract TransferOwnerTemplate is MultisigTask {
     address public newOwner;
 
     /// @notice Returns the safe address string identifier
-    /// @return The string "SystemConfigOwner"
+    /// @return The string "ProxyAdminOwner"
     function safeAddressString() public pure override returns (string memory) {
-        return "SystemConfigOwner";
+        return "ProxyAdminOwner";
```

Comment:
Why was this change made? Seems like a big semantic change to this template

Reply:
Isn't this the wrong address string being used here? Transferring ownership on the ProxyAdmin should be by the ProxyAdminOwner.

Reply:
ah I now see the natspec here:

> Template contract for transferring ownership of the proxy admin

So then yes I agree, resolving